src/Stratis.Bitcoin.Tests/Consensus/ChainedHeaderTreeTest.cs

```diff
@@ -2252,6 +2252,62 @@ private void CheckChainedHeaderTreeConsistency(ChainedHeaderTree cht, TestContex
             Assert.Equal(chainHeaders.Count, allConnectedHeaders.Count);
         }
 
+         /// <summary>
+        /// Issue 28 @ Peer E,F claims 11d. PartialOrFullValidationFailed(7a), make sure that 7a,8a,9a,8d,9d,10d,11d are removed and the peers A E F D are marked as PeersToBan.
```

Comment:
In the implementation below you have decided for those two peers to claim different headers, which is fine considering what we are testing, so just maybe Modifie description here that peer E claims 11e and peer F claims 11f and in the second part of the description instead of 11d mention both 11e and 11f

Reply:
The description is actually bit incorrect. It's should be Peers E, F claims 10d. Cause in initial setup 11d doesn't exist.
I will fix it.